k9mail-library/src/main/java/com/fsck/k9/mail/store/imap/ImapStore.java

```diff
@@ -77,7 +77,7 @@
  * </pre>
  */
 public class ImapStore extends RemoteStore {
-    public static final String STORE_TYPE = "IMAP";
+    public static final ServerSettings.Type STORE_TYPE = ServerSettings.Type.IMAP;
```

Comment:
With the introduction of the enum(s) the constants in the store classes should be removed.